website/source/docs/providers/aws/r/route.html.markdown

```diff
@@ -35,12 +35,14 @@ The following arguments are supported:
 * `destination_cidr_block` - (Required) The destination CIDR block.
 * `vpc_peering_connection_id` - (Optional) An ID of a VPC peering connection.
 * `gateway_id` - (Optional) An ID of a VPC internet gateway or a virtual private gateway.
```

Comment:
My opinion here is that it is better to keep these separate - otherwise we'd have to parse the ID to work out how to make the API call.